File: Lista V (C++) - Aula 22/C1_02/Funcionario.cpp
#include <iostream>
#include "Departamento.h"

//Construtores
Funcionario::Funcionario(const string& nome, double salario, const string& dataAdmissao) {
	this->setNome(nome);
	this->setSalario(salario);
	this->setDataAdmissao(dataAdmissao);
}

//GETs & SETs

void Funcionario::setNome(const string& nome){
	this->nome = nome;
}

double Funcionario::getSalario() const {
	return salario;
}
void Funcionario::setSalario(double salario){
	this->salario = salario;
}

void Funcionario::setDataAdmissao(const string& dataAdmissao){
	this->dataAdmissao = dataAdmissao;
}

//Métodos
void Funcionario::darAumento(double percentual) {
	salario += salario * percentual / 100;
}

File: Lista II (JAVA) - Aula 4/J1_08/Funcionario.java
public class Funcionario {
    String nome;
    double salario;
    String data;

    public void setNome(String nome) {
        this.nome = nome;
    }

    String getNome(){
        return this.nome;
    }

    public void setSalario(double salario) {
        this.salario = salario;
    }

    double getSalario(){
        return this.salario;
    }

    public void setData(String data) {
        this.data = data;
    }

    String getData(){
        return this.data;
    }
}
